examples/librimix/librimix.ipynb

```diff
@@ -124,10 +125,43 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "clean_cut_set = mix_by_recording_id(cut_sets=[librimix_manifests['sources']['cuts']])\n",
```

Comment:
a few lines above here, I don't understand why you are doing `cuts[1:], cuts[0]`... it's like you are moving the 1st cut to the end?

Reply:
also on that line, a I think a line break before cuts[1:] would make it more readable.

Reply:
Yeah, it's unnecessary. I thought that `reduce` requires an explicit starting value, but it turns out it doesn't. I simplified the usage in most places but missed this one.

Reply:
Now I think that's because of using C++ `std::accumulate` too much in the past...

Reply:
ha ha, OK